src/gui/MItem_menus.cpp

```diff
@@ -402,17 +402,37 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
     : WI_LABEL_t(_(label), nullptr, is_enabled_t::yes, is_hidden_t::no) {
 }
 
+struct SettingsLoadState {
+    bool is_network;
```

Comment:
Naming the variables `is_` is very confusing here. Much better would be `network_settings_loaded` and `connect_settings_loaded`.

Also, you can make these use a single bit to save space by `bool network_settings_loaded : 1;`

Reply:
Also, as @vorner previously remarked, you don't really need a structure here at all. You would simply do with two local variables inside MI_LOAD_SETTINGS::click.

Reply:
Done, removed the struct